### DISCUSSION TITLE
Option --manifests

### DISCUSSION THREAD

**Discussion by kondil** [fluxcd/flux2#4698] [General] [answered]
Hey,

I am using flux cli `v2.1.2`.

Recently a colleague showed me the bellow option

```sh
flux install --manifests /path/to/flux_repo/manifests
```

When I looked into [flux install](https://fluxcd.io/flux/cmd/flux_install/) or the help page of the command

```sh
flux install --help | grep mani
```

```sh
  # Write install manifests to file
      --export                     write the install manifests to stdout and exit
  -v, --version string             toolkit version, when specified the manifests are downloaded from https://github.com/fluxcd/flux2/releases
```

I couldn't find any result of this specific option.

Is there something I am missing, or is this just an undocumented option?

**Comment by stefanprodan** [comment, marked as answer]
This option is for internal testing only, using it in production is not recommended as none of the flags will work with it.

**Reply by kondil** [reply]
Thank you very much for your quick confirmation!
